### DISCUSSION TITLE
Email + Sign-in provider: Microsoft 📬

### DISCUSSION THREAD

**Discussion by FelixMalfait** [twentyhq/twenty#3940] [Ideas]
We've enabled sign-in though Google but it would be nice to also support for Microsoft Office 365.

**Comment by stereo9** [comment]
i support this, generally oidc / oauth to open the app up do identity providers such as keycloak would be great

**Comment by SamAmann** [comment]
It would be a game-changer for my company (and probably more) to be able to link M365 accounts! (and probably more with any SMTP/IMAP)

**Comment by linjoe2** [comment]
general OIDC to me is an apsolute must to make it work within my current infrastructure, if any of the code editors could give me some pointers into the right direction, I'd love to spend some hours on development.

**Comment by FelixMalfait** [comment]
Coming in November: https://github.com/twentyhq/twenty/pull/8044 :)

**Comment by Bonapara** [comment]
Online!

https://twenty.com/releases
